package.json

```diff
@@ -13,44 +13,44 @@
     "test": "mocha build/server/serving_test.js"
   },
   "dependencies": {
-    "@google-cloud/debug-agent": "^1.0.0",
-    "@google-cloud/logging": "^1.0.0",
-    "@google-cloud/trace-agent": "^1.0.3",
-    "@types/cors": "^2.8.3",
-    "body-parser": "^1.17.1",
-    "compression": "^1.6.2",
-    "concurrently": "^3.4.0",
-    "connect-memcached": "^0.2.0",
-    "cookie-parser": "^1.4.3",
-    "core-js": "^2.4.1",
-    "cors": "^2.8.4",
-    "express": "^4.15.0",
+    "@google-cloud/debug-agent": "^4.0.3",
+    "@google-cloud/logging": "^5.3.1",
+    "@google-cloud/trace-agent": "^4.2.0",
+    "@types/cors": "^2.8.6",
+    "body-parser": "^1.19.0",
+    "compression": "^1.7.4",
+    "concurrently": "^4.1.2",
+    "connect-memcached": "^1.0.0",
+    "cookie-parser": "^1.4.4",
+    "core-js": "^3.2.1",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
     "express-enforces-ssl": "^1.1.0",
-    "express-recaptcha": "^2.1.0",
-    "express-session": "^1.15.1",
-    "form-data": "^2.1.2",
-    "googleapis": "^17.0.0",
-    "helmet": "^3.4.1",
-    "morgan": "^1.8.1",
-    "passport": "^0.3.2",
-    "passport-google-oauth20": "^1.0.0",
-    "recaptcha2": "^1.0.8",
-    "request": "^2.78.0",
-    "serve-favicon": "^2.3.0"
+    "express-recaptcha": "^5.0.1",
+    "express-session": "^1.16.2",
+    "form-data": "^2.5.1",
+    "googleapis": "^43.0.0",
+    "helmet": "^3.21.1",
+    "morgan": "^1.9.1",
+    "passport": "^0.4.0",
+    "passport-google-oauth20": "^2.0.0",
+    "recaptcha2": "^1.3.3",
+    "request": "^2.88.0",
+    "serve-favicon": "^2.5.0"
   },
   "devDependencies": {
-    "@types/body-parser": "^1.16.3",
-    "@types/compression": "^0.0.33",
-    "@types/express-session": "^0.0.32",
-    "@types/helmet": "^0.0.33",
-    "@types/mocha": "^2.2.41",
-    "@types/supertest": "^2.0.0",
-    "chai": "^3.5.0",
-    "jasmine": "^2.5.3",
-    "supertest": "^3.0.0",
-    "tslint": "^4.5.1",
-    "typedoc": "^0.5.7",
-    "typescript": "^2.2.1",
+    "@types/body-parser": "^1.17.1",
+    "@types/compression": "^1.0.1",
+    "@types/express-session": "^1.15.14",
+    "@types/helmet": "^0.0.44",
+    "@types/mocha": "^5.2.7",
+    "@types/supertest": "^2.0.8",
+    "chai": "^4.2.0",
+    "jasmine": "^3.5.0",
+    "supertest": "^4.0.2",
+    "tslint": "^5.20.0",
+    "typedoc": "^0.15.0",
+    "typescript": "<=3.5",
```

Comment:
I think you can just specify something like ^3.5.0 or the equivalent, which is a narrower range than this and better for build dependencies. Other libraries you're changing here are linked to specific typescript versions so it's better to define the scope as narrowly as possible.

Reply:
Ah, makes sense. I specified to ^3.5.3, which was the latest version before 3.6.